cmd/mockery.go

```diff
@@ -91,6 +91,7 @@ func init() {
 	pFlags.BoolP("dry-run", "d", false, "Do a dry run, don't modify any files")
 	pFlags.Bool("disable-version-string", false, "Do not insert the version string into the generated mock file.")
 	pFlags.String("boilerplate-file", "", "File to read a boilerplate text from. Text should be a go block comment, i.e. /* ... */")
+	pFlags.Bool("vararg", false, "Generates mocks with vararg as a single argument.")
```

Comment:
I think better terminology here would be to say "For functions with variadic arguments, do not unroll the arguments into the underlying testify call. Instead, pass variadic slice as-is."

As for the name, I think it makes more sense to call if `--no-unroll-variadic`. It's more verbose but is clearer what it does. Thoughts?

Reply:
I like `--unroll-variadic`. Cons: it will be used in the project with negation only `if !g.UnrollVariadic ...`, which I think is reasonable cost for the self-explanatory argument.
If you are ok with it I'll make the adjustments.

Reply:
Yeah let's go ahead with that, I think it will make more sense. Thanks for the time you're putting into this!

Reply:
thx for the project!